src/main/java/ledger/user_interface/ui_controllers/MainPageController.java

```diff
@@ -150,7 +150,8 @@ private void createAccountPopup() {
      */
     private void deleteAccount() {
         if (chooseAccount.getSelectedAccount() == null) {
-            setupErrorPopup("Cannot delete the All Accounts aggregation", new Exception());
+            setupErrorPopup("Cannot delete the All Accounts aggregation. Please select an account " +
+                    "to delete.", new Exception());
```

Comment:
Nope, we dont use the stack trace here. It was originally just a hacky solution to get around emily's setupErrorPopup. I can write a version that doesnt take an exception later tonight